cli/githubutil.py

```diff
@@ -82,4 +86,45 @@ def release_and_prepare_next_dev_cycle(
         commit_message="Prepare next dev cycle " + next_version_dev
     )
 
+def remove_webhooks(
+    github_org_name: str,
+    github_cfg_name: str,
+    concourse_cfg_name: str,
+):
+    '''
+    Remove all webhooks which belong to the given concourse_cfg_name
+    '''
+    not_empty(github_org_name)
+    not_empty(github_cfg_name)
+    not_empty(concourse_cfg_name)
+
+    cfg_factory = ctx().cfg_factory()
+    github_cfg = cfg_factory.github(github_cfg_name)
+
+    github_api = _create_github_api_object(github_cfg=github_cfg)
+    github_org = github_api.organization(github_org_name)
+    webhook_syncer = GithubWebHookSyncer(github_api)
+
+    def filter_function(url):
+        concourse_id = parse_qs(urlparse(url).query).get('concourse_id')
```

Comment:
consider moving concourse_id to a "define" in github/__init__.py to avoid duplication

Reply:
also consider either commenting what True/False mean or just assign the value to an expressively-named variable (e.g. should_delete=...)